setup.py

```diff
@@ -245,8 +245,8 @@ def run(self):
     'package_data'       : {'': ['*.txt', '*.sh', '*.json', '*.gz', '*.c',
                                  '*.md', 'VERSION', 'SDIST', sdist_name]},
   # 'setup_requires'     : ['pytest-runner'],
-    'install_requires'   : ['radical.utils>=0.90',
-                            'radical.saga>=0.90',
+    'install_requires'   : ['radical.utils>=1.1',
+                            'radical.saga>=1.0',
```

Comment:
Shouldn't this be 1.1 also?

Reply:
1.0 will work - there was no change in RS which RP depends on between 1.0 and 1.1.